lib/consumer.ex

```diff
@@ -99,7 +99,8 @@ defmodule GenRMQ.Consumer do
         durable: true,
         passive: true,
         arguments: [
-          {"x-queue-type", :longstr ,"quorum"},
+          {"x-queue-type", :longstr ,"quorum"}
+        ]
```

Comment:
Is this change meant to be in this PR?

Reply:
Yes, this are the code docs and there was a typo.